Sources/Xcodeproj/SchemesGenerator.swift

```diff
@@ -86,7 +86,7 @@ public final class SchemesGenerator {
             }
         })
         schemes.append(Scheme(
-            name: rootPackage.name + "-Package",
+            name: rootPackage.manifestName + "-Package", // TODO: use identity instead?
```

Comment:
Yes, probably everywhere name is used we should start using identity.

Reply:
https://github.com/apple/swift-package-manager/pull/3323#discussion_r589835348